app/views/dashboard/index.html.erb

```diff
@@ -1,3 +1,38 @@
-<%= t('defaults.welcome') %>, <%= current_user.first_name %> !
-<br/>
-<%= link_to t('.choose_my_courses'), new_dashboard_subscription_path, class: "btn btn-primary" %>
+<div class="container">
+  <%= t('defaults.welcome') %>, <%= current_user.first_name %> !
+  <br/>
+  <% if @subscription.present? %>
+    <p>Voici les informations detaillés de votre inscription pour l'ané <%= @subscription.year %>:</p>
```

Comment:
Voici les informations detaillées de votre inscription pour l'année <%= @subscription.year %>:
```suggestion
    <p>Voici les informations detaillées de votre inscription pour l'année <%= @subscription.year %>:</p>
```

Even better:
```suggestion
    <p><%=t('.subscription_detailed_info', year: @subscription.year) %></p>
```

And in `fr.yml`
```yaml
dashboard:
  index:
    subscription_detailed_info: "Voici les informations detaillées de votre inscription pour l'année %{year}"
```